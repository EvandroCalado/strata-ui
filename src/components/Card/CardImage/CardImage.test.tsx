import { render, screen } from '@testing-library/react';
import { CardImage } from '.';

describe('<CardImage />', () => {
  it('should render a CardImage by default', () => {
    render(<CardImage src="test.png" />);

    expect(screen.getByRole('img')).toBeInTheDocument();
    expect(screen.getByRole('img')).toHaveAttribute('src', 'test.png');
  });

  it('should render a snapshot', () => {
    const { container } = render(<CardImage src="test.png" />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
