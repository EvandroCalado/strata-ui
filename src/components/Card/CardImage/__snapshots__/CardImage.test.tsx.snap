// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<CardImage /> > should render a snapshot 1`] = `
<img
  class=""
  src="test.png"
/>
`;
